scripts/enterGame.js

```diff
@@ -1,7 +1,7 @@
 // establish the elements that will be edited
-let title = document.getElementById("title").innerHTML;
-let start = document.getElementById("start").innerHTML;
-let draw = document.getElementById("draw").innerHTML;
+let titleElement = document.getElementById("title");
```

Comment:
if you are referencing an element, add `element` to the suffix to make it more clear.
Also you want the element object, and not the `innerHTML`